src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -5,30 +5,101 @@ import {
   GridColumns,
   Separator,
   Text,
+  useToasts,
 } from "@artsy/palette"
-import { Formik } from "formik"
+import { isEmpty, camelCase, snakeCase } from "lodash"
+import { FC } from "react"
+import { Form, Formik } from "formik"
+import { createFragmentContainer, graphql } from "react-relay"
+import {
+  PreferencesApp_viewer,
+  SubGroupStatus,
+} from "v2/__generated__/PreferencesApp_viewer.graphql"
+import { useEditNotificationPreferences } from "./useEditNotificationPreferences"
+
+const NOTIFICATION_FIELDS = {
+  recommendedByArtsy: false,
+  artWorldInsights: false,
+  productUpdates: false,
+  guidanceOnCollecting: false,
+  customAlerts: false,
+}
+
+interface PreferencesAppProps {
+  viewer?: PreferencesApp_viewer
+}
+
+interface FormValuesForNotificationPreferences {
+  recommendedByArtsy: boolean
+  artWorldInsights: boolean
+  productUpdates: boolean
+  guidanceOnCollecting: boolean
+  customAlerts: boolean
+}
+
+const parseViewer = viewer => {
+  return viewer?.notificationPreferences
+    .filter(preference =>
+      Object.keys(NOTIFICATION_FIELDS).includes(camelCase(preference.name))
+    )
+    .reduce((object, preference) => {
+      object[camelCase(preference.name)] =
+        preference.status === "SUBSCRIBED" ? true : false
+      return object
+    }, {})
+}
+
+export const PreferencesApp: FC<PreferencesAppProps> = ({ viewer }) => {
+  const { sendToast } = useToasts()
+  const { submitMutation } = useEditNotificationPreferences()
+  let initialValues = parseViewer(viewer) // Shape the response from Metaphysics for Formik
```

Comment:
something like 
```
const initialValues = getInitialValues(viewer) 
```
would help hide some of the implementation details. I'd also move `getInitialValues` down to the bottom of the component since that's secondary to the component itself, for future devs.